src/westmere/sse_convert_latin1_to_utf32.cpp

```diff
@@ -0,0 +1,147 @@
+/* Par/scalar benchmark:
```

Comment:
@Nick-Nuon Can you clean up this file so we can consider merging it?

(It is fine to have long comments if you'd like, but make sure that they are formatted nicely so that it is readable.)

Reply:
Done!

Reply:
Done!

Reply:
I saw the fuzzer test was failing, on it today.

Reply:
Good work!